.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -38,17 +38,20 @@ The following known issues need to be reviewed in case a compatible version is a
 
 ### Dockerfile updates
 Update versions in the following dockerfiles
-- [ ] `./baseDockerfiles/cypressBaseImageDockerfile`
-  - [ ] On any change run the corresponding job in GitHub
-- [ ] `./baseDockerfiles/temurinBaseImageDockerfile`
-  - [ ] On any change run the corresponding job in GitHub
+- [ ] `./dataland-backend/DatalandBackendBaseDockerfile`
 - [ ] `./dataland-backend/Dockerfile`
+- [ ] `./dataland-backend/DockerfileTest`
+- [ ] `./dataland-e2etests/e2etestsCoreDockerfile`
+- [ ] `./dataland-frontend/Dockerfile`
+- [ ] `./dataland-frontend/DockerfileTest`
+- [ ] `./dataland-inbound-admin-proxy/Dockerfile`
+- [ ] `./dataland-inbound-proxy/DockerfileBase`
+- [ ] `./dataland-inbound-proxy/Dockerfile`
+- [ ] `./dataland-keycloak/Dockerfile`
```

Comment:
das Keycloak docker file taucht in der Liste doppelt auf

Reply:
removed